docs/content/en/docs/getting-started/optional/irsa.md

```diff
@@ -117,8 +117,9 @@ Set the remaining fields in cluster spec as required and create the cluster usin
 
 #### Generate keys.json and make it publicly accessible
 
-1. The cluster provisioning workflow generates a pair of service account signing keys. Retrieve the public signing key generated and used by the cluster, and create a keys.json document containing the public signing key. 
-
+1. The cluster provisioning workflow generates a pair of service account signing keys. Retrieve the public signing key generated and used by the cluster, and create a keys.json document containing the public signing key.
+2. We need minimum go version 1.19 to run it . Use  `go version` command to check installed Go version.
```

Comment:
Is there somewhere in the amazon-eks-pod-identity-webhook they can get the go version?

Reply:
No , go version is not mentioned anywhere , that's the reason i am adding here so that people install correct version

Reply:
I will update  in the amazon-eks-pod-identity-webhook also , but IRSA itself is a complicated process , so i would suggest to mention here as well as a perequistre

Reply:
Maybe have  users look to https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/go.mod#L3

Reply:
Infrastructure people who manage onpremise server and responsible for setup EKS-A typically don't see this code because only few people are familiar with the go programming language. 

I wanted to edit this here as well because of that, but if you still think it's not necessary, i'll remove that part.

Reply:
@TerryHowe Reverted that changes , please have a look